documentation/book/proc-creating-new-image-from-base.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// assembly-using-kafka-connect-with-plugins.adoc
+
+include::common/attributes.adoc[]
+
+[id='creating-new-image-from-base-{context}']
+= Create a new image based on our base image
+
+{ProductName} provides its own Docker image for running Kafka Connect, which can be found on {DockerRepository} as
+`{DockerKafkaConnect}`.
+This image can be used as a base image for building a new custom image with additional plugins.
+
+The following procedure describes the process for creating such a custom image.
+
+.Procedure
+
+. Create a new `Dockerfile` using `{DockerKafkaConnect}` as the base image
++
+[source,Dockerfile,subs="attributes"]
+----
+FROM {DockerKafkaConnect}
+USER root:root
+COPY ./my-plugin/ /opt/kafka/plugins/
+USER {DockerImageUser}
+----
+
+. Build the Docker image and upload it to the appropriate Docker repository
+
+. Use the new Docker image in the Kafka Connect deployment:
+.. On {OpenShiftName}, the template parameters `IMAGE_REPO_NAME`, `IMAGE_NAME`, and `IMAGE_TAG` can be changed to point to the new image when the Kafka Connect cluster is being deployed.
```

Comment:
These parameters have been removed from the templates. We should instead link to the `KafkaConnect.spec.image` property of the Kafka Connect custom resource ([section 3.2.2](http://strimzi.io/docs/master/#kafka_connect_config_map_details)) and to the `STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE` variable ([section 3.4](http://strimzi.io/docs/master/#STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE)).